editor/sidebar/block-inspector/index.js

```diff
@@ -14,17 +14,18 @@ import { Panel, PanelBody } from 'components';
  * Internal Dependencies
  */
 import './style.scss';
-import { deselectBlock } from '../../actions';
+import BlockInspectorClassName from './class-name';
 import { getSelectedBlock } from '../../selectors';
 
-const BlockInspector = ( { selectedBlock } ) => {
-	if ( ! selectedBlock ) {
+const BlockInspector = ( { hasSelectedBlock } ) => {
+	if ( ! hasSelectedBlock ) {
 		return <span className="editor-block-inspector__no-blocks">{ __( 'No block selected.' ) }</span>;
 	}
 
 	return (
 		<Panel>
 			<PanelBody>
+				<BlockInspectorClassName />
```

Comment:
This should be rendered after the slot, and probably include a separator (`hr`).